src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
+    public double calculateArea();
```

Comment:
```suggestion
    double calculateArea();
```
methods in an interface are public abstract by default